core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20190805_add_billing_project_location.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="dummy" author="mbemis" id="location-on-billing-projects">
+        <addColumn tableName="BILLING_PROJECT">
+            <column name="LOCATION" type="VARCHAR(255)">
+                <constraints nullable="true"/>
+            </column>
+        </addColumn>
+    </changeSet>
+
+    <changeSet logicalFilePath="dummy" author="mbemis" id="backfill-project-locations">
+        <sql stripComments="true">
+            update BILLING_PROJECT set LOCATION="us" where LOCATION=NULL
```

Comment:
I don't think =NULL works in sql (it's always false), you have to do `where LOCATION IS NULL`

Reply:
but +100 for backfilling 🥇

Reply:
oops, hadn't actually run this particular changeset yet 😆 

updated